### PR TITLE
Add club partners

### LINKED ISSUE #98: Add partner club logos
As multiple clubs are hosting workshops at our event, we should add their logos to the website.

### PR DESCRIPTION
**Pull request by samderanova**
- Add logos for clubs that are hosting workshops, each with their own `alt` tags and links as specified in `src/assets/data/clubs.json`
- Create new clubs section component

### CODE REVIEW

**Review by lassenordahl — DISMISSED** [review]
:lgtm:

**Review by njhuey — DISMISSED** [review]

**Review by rileysw — DISMISSED** [review]
May look awkward with the current sponsors layout (blue background). Everything else looks good!

File: src/views/home/sections/Clubs/Clubs.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+		{ png: design, info: clubs[4] },
+		{ png: icssc, info: clubs[5] },
+		{ png: data, info: clubs[6] },
+	],
+];
+
+function Clubs() {
+	return (
+		<Container className="p-5">
+			<h2>Partners</h2>
```
Comment (taesungh) [inline src/views/home/sections/Clubs/Clubs.tsx:46]: Nitpick (non-blocking): component name should match section name

File: src/views/home/sections/Clubs/Clubs.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+					className="my-3 justify-content-center align-items-center"
+				>
+					{row.map((club, imgIndex) => (
+						<Col sm key={imgIndex} className="px-5 my-3">
+							<a href={club.info.clubWebsite}>
+								<Image
+									key={imgIndex}
+									src={club.png}
+									className={"w-100 h-auto " + styles["hover-enlarge"]}
+									alt={`${club.info.name} club logo`}
```
Comment (taesungh) [inline src/views/home/sections/Clubs/Clubs.tsx:63]: Nitpick (non-blocking): Don't think ASUCI can be described as a club.

File: src/views/home/sections/Clubs/Clubs.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+			<h2>Partners</h2>
+			<p className="text-center">
+				Thank you to our partners for hosting workshops at our events!
+			</p>
+
+			{CLUBS.map((row, index) => (
+				<Row
+					key={index}
+					className="my-3 justify-content-center align-items-center"
+				>
```
Comment (taesungh) [inline src/views/home/sections/Clubs/Clubs.tsx:55]: Note: if partner names are unique, the name could be used as a key rather than the index.

**Review by taesungh — DISMISSED** [review]
Thanks for the quick work on this, minor nitpicks, but looks alright.

**Review by taesungh — APPROVED** [review]

**Review by njhuey — APPROVED** [review]

**Pull request merged**